gst/parser/gstttmlparse.cpp

```diff
@@ -906,7 +907,7 @@ parse_subrip (ParserState * state, const gchar * line)
       gchar *end_time;
 
       /* looking for start_time --> end_time */
-      if ((end_time = strstr (line, " --> ")) &&
+      if ((end_time = g_strdup (strstr (line, " --> "))) &&
```

Comment:
Are you sure about this `g_strdup`? Looks like a leak...

Reply:
@roman380 I think it doesn't really matter, because this change was made due to C->C++ transfer to compile code. It'll be reverted back as soon as we rename files to *.c back.